src/components/Comments/CommentForm/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { ChangeEventHandler } from "react";
+
+import { Send as SendIcon } from "@material-ui/icons";
+
+import BoxBase from "../../Boxs/BoxBase";
+import ButtonBase from "../../Buttons/ButtonBase";
+import TextFieldBase from "../../Textfields/TextFieldBase";
+
+export interface CommentFormProps {
+    isLoading?: boolean;
+    value?: string;
+    submit: (content: string) => void;
+}
+
+const CommentForm = (props: CommentFormProps) => {
+    const { isLoading, value, submit } = props;
+    const [content, setContent] = useState<string>(value ? value : "");
+
+    const changeHandler: ChangeEventHandler<HTMLTextAreaElement> = (event) => {
+        setContent(event.target.value);
+    };
+
+    const submitHandler = () => {
+        submit(content);
+        setContent("");

```

Comment:
I don't understand

Reply:
I created submitHandler instead of using submit function directly because I need to set the content state to empty string.